src/shared.ts

```diff
@@ -27,7 +30,11 @@ const componentNameMapping: Record<
   "payment-method-settings": "stripe-connect-payment-method-settings",
   "account-management": "stripe-connect-account-management",
   "notification-banner": "stripe-connect-notification-banner",
-  "instant-payouts": "stripe-connect-instant-payouts"
+  "instant-payouts": "stripe-connect-instant-payouts",
+};
+
+type StripeConnectInstanceExtended = StripeConnectInstance & {
+  debugInstance: () => Promise<StripeConnectInstance | undefined>;
```

Comment:
This hasn't yet been implemented in connect.js right?

Reply:
No, we would wait on this to be published first before making those changes right? since we need the new package version